phoenix-core/src/test/java/org/apache/phoenix/compile/WhereCompilerTest.java

```diff
@@ -711,6 +714,16 @@ public void testOr2InFilter() throws SQLException {
         assertArrayEquals(ByteUtil.nextKey(stopRow), scan.getStopRow());
     }
 
+    @Test
```

Comment:
Ignore this test and I forgot to remove it. It will be removed in the next commit.